### PR TITLE
add id searching to extension list

### PR DESCRIPTION
**Pull request by Seishirou101**
adds ID searching and baseurl. Not sure if the self hosted one's baseurl should be removed as that might cause confusion to *some* people but who knows.

Also not sure if we should do hyperlinks? but that can easily be changed if its not wanted.

### CODE REVIEW

File: src/.vuepress/components/Extensions.vue [hunk 1]
```diff
@@ -42,6 +43,11 @@ export default {
 						ext.name.toLowerCase().includes(filters.search.toLowerCase())
 					);
 				}
+				if (filters.searchID) {
+					filteredGroup = filteredGroup.filter((ext) =>
+						ext.sources.some((idnum) => idnum.id.includes(filters.searchID))
+					);
+				}
```
Comment (ghostbear) [inline src/.vuepress/components/Extensions.vue:50]: Pretty sure the search field could be reused for this

```js
if (filters.search) {
  filteredGroup = filteredGroup.filter((ext) =>
    ext.name.toLowerCase().includes(filters.search.toLowerCase()) ||
    ext.sources.some((idnum) => idnum.id.includes(filters.searchID))
  );
}
```

Comment (Seishirou101) [reply src/.vuepress/components/Extensions.vue:50]: I initially did this but for some reason thought they may need to filter out names on top of ID... which now makes no sense. will change it o7

**Review by ghostbear — APPROVED** [review]
I'm fine with the filter changes. 

But I'm not sure about adding `baseUrl`.

**Comment by arkon** [comment]
I'd personally prefer not readily exposing the links like that.

**Comment by Seishirou101** [comment]
> 
> 
> I'd personally prefer not readily exposing the links like that.

okay, removed

**Review by arkon — APPROVED** [review]

**Pull request merged**